styles/lib/buttons.css

```diff
@@ -6,8 +6,9 @@ button {
     border-radius: var(--radius);
     padding: var(--padding);
     box-sizing: border-box;
-    height: 2rem;
+    height: 2.5rem;
     min-width: 2rem;
+    font-weight: 700;
```

Comment:
font weight needs to be variable. do we have set amount of font weights from dancus?

Reply:
added a bold variable to the text css file